testbed/tests/widgets/test_base.py

```diff
@@ -69,7 +69,9 @@ async def test_visibility(widget, probe):
     assert child_probe.is_hidden
     assert grandchild_probe.is_hidden
 
-    # Mark the child style as visible.
+    # Mark the child style as visible. Mark it as hidden first, so that a change is
+    # registered and the property is applied.
```

Comment:
It took me a second to mentally validate this was correct; suggest extending the explanation to make it clear that this is a no-op - we're just *confirming* it's a no-op.
```suggestion
    # Mark the child style as visible. Mark it as hidden first, so that a change is
    # registered and the property is applied. However, regardless, the widget 
    # won't be visible because it has an ancestor that is is hidden.
```

Reply:
Entirely reasonable! I pushed my own commit just because you had a double "is", but yes, this is the case, and saying so is a good idea.